guides/source/active_record_querying.md

```diff
@@ -2294,31 +2294,18 @@ irb> nina.save
 Finding by SQL
```

Comment:
Im tempted to say that we should not have a section for `find_by_sql` in this document at all.

Its absolutely bizarre when people start selecting columns outside of the model you are working with, which is what commonly happens here.

In place of I would suggest a section in `select_all` where it recommends that any usage of `find_by_sql` might be better served by the `select_all` method instead.